match/tangipahoa_da.py

```diff
@@ -0,0 +1,36 @@
+import sys
+sys.path.append('../')
+from datamatch import JaroWinklerSimilarity, ThresholdMatcher, ColumnsIndex
+from lib.path import data_file_path
+import pandas as pd 
+
+
+def match_cprr_with_post(cprr, post):
+    dfa = cprr[['first_name', 'last_name', 'uid', 'agency']]
+    dfa.loc[:, 'fc'] = dfa.first_name.map(lambda x: x[:1])
+    dfa = dfa.drop_duplicates(subset=['uid']).set_index('uid')
+
+    dfb = post[['first_name', 'last_name', 'uid', 'agency']]
```

Comment:
Don't you need to filter post data down to an agency first?

Reply:
Not yet, because most of them belong to distinct agencies, and, at the moment, I don't know what those agencies are. I was just exploring. I will filter by the agency once I have an agency for each officer. 

That said, if I'm unable to acquire the corresponding agency for each officer I think that I will stick with attempting to match to the entire post dataset.